source/_components/mychevy.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
+title: "MyChevy"
+description: "Instructions on how to integrate Chevy Bolt car into Home Assistant."
+date: 2017-08-28 17:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: tesla.png
+ha_category: Hub
+ha_release: 0.61
+ha_iot_class: "Cloud Polling"
+---
+
+The `MyChevy` component communicates with the
+[my.chevrolet](https://my.chevrolet.com) website using selenium to log
+in as a your user, and screen scrape the data provided. GM does not
```

Comment:
`in as a your user`  -> `in as your user`

Reply:
ack